node/rest/src/routes.rs

```diff
@@ -177,36 +177,6 @@ impl<N: Network, C: ConsensusStorage<N>> Rest<N, C> {
             .and(with(self.ledger.clone()))
             .and_then(Self::find_transition_id);
 
-        // GET /testnet3/records/all
-        let records_all = warp::get()
-            .and(warp::path!("testnet3" / "records" / "all"))
-            .and(with_auth())
```

Comment:
I believe these endpoints were the only ones using the `with_auth` from https://github.com/AleoHQ/snarkOS/blob/testnet3/node/rest/src/helpers/auth.rs . That file as well as the code here: https://github.com/AleoHQ/snarkOS/blob/testnet3/node/rest/src/lib.rs#L118 can probably be deleted.

Reply:
Agreed, we discussed and decided to keep for guarding the existing REST endpoints prior to mainnet.